docs/src/pages/guides/FHIRBulkOperations.md

```diff
@@ -0,0 +1,193 @@
+---
+layout: post
+title: FHIR Bulk Data Guide
+description: FHIR Bulk Data Guide
+date:   2020-05-08 09:00:00 -0500
+permalink: /FHIRBulkOperations/
+---
+
+# Overview
+
+The IBM FHIR Server has extended operations for Bulk Data `$import` and `$export`, which are implemented in the modules: 
+
+|Module|Description|
+|---|---|
+|[fhir-operation-bulkdata](https://github.com/IBM/FHIR/tree/master/fhir-operation-bulkdata)|Implements the FHIR Operations `$import` and `$export` and translate bulk data requests into JSR352 Java Batch jobs|
+|[fhir-bulkimportexport-webapp](https://github.com/IBM/FHIR/tree/master/fhir-bulkimportexport-webapp)|Standalone web application to process bulk data requests as JSR352 Java Batch jobs|
+
+The IBM FHIR Server bulk data module configuration is described in more detail at the [FHIR Server Users Guide](https://ibm.github.io/FHIR/guides/FHIRServerUsersGuide/#410-bulk-data-operations).
+
+## Export Operation: $export
+The `$export` operation uses three OperationDefinition: 
+- [System](http://hl7.org/fhir/uv/bulkdata/STU1/OperationDefinition-export.html) - Export data from the server. Exports to an S3-compatible data store.
+- [Patient](http://hl7.org/fhir/uv/bulkdata/STU1/OperationDefinition-patient-export.html) - An instance level execution, Obtain a set of resources pertaining to all patients. exports to an S3-compatible data store.
+- [Group](http://hl7.org/fhir/uv/bulkdata/STU1/ OperationDefinition-group-export.html) - Obtain a set resources pertaining to patients in a specific Group. Only supports static membership; does not resolve inclusion/exclusion criteria.
+
+### **$export: Create a Bulk Data Request**
+To create an import request, the IBM FHIR Server requires the body fields of the request object to be a FHIR Resource `Parameters` JSON Object.  The request must be posted to the server using `POST`. Each request is limited to a single resource type in each imported or referenced file.
+
+#### Example Request - GET
+The following is a request to export data to the IBM COS endpoint from the IBM FHIR Server using GET.
+
+```sh
+curl -u "fhiruser:change-password" -k -H "Content-Type: application/fhir+json" -X GET 'https://localhost:9443/fhir-server/api/v4/$export?_outputFormat=application/fhir%2Bndjson&_type=Patient' -v
```

Comment:
totally just a preference thing, but i prefer the `-k` to be the first flag so its more prominent (since its not great practice...but so handy)
```suggestion
curl -k -u "fhiruser:change-password" -H "Content-Type: application/fhir+json" -X GET 'https://localhost:9443/fhir-server/api/v4/$export?_outputFormat=application/fhir%2Bndjson&_type=Patient' -v
```
Totally feel free to close/ignore this.